packages/shared/src/components/modals/VideoModal.tsx

```diff
@@ -26,10 +25,9 @@ export default function VideoModal({
       {...props}
     >
       <CloseButton
```

Comment:
Should this not be `ModalCloseButton` then?
(it's not really blocking tbh

Reply:
That's a good question. Not sure why, maybe again something from the past that we can now change to be consistent. Let me try...

Reply:
OK, I refactored to `ModalClose` where it made sense (in Modals mostly). Please take a look again 🙏